flytekit/common/tasks/sdk_dynamic.py

```diff
@@ -305,3 +248,65 @@ def execute(self, context, inputs):
             })
 
             return generated_files
+
+
+class SdkDynamicTask(_six.with_metaclass(_sdk_bases.ExtendedSdkType, SdkDynamicTaskMixin, _sdk_runnable.SdkRunnableTask)):
```

Comment:
we can stop using six - in the refactor that we're doing we're clearing out a lot of the python2 stuff.  feel free to just `metaclass= _sdk_bases.ExtendedSdkType`